kinds/structured/preferences/preferences_kind.cue

```diff
@@ -0,0 +1,49 @@
+package kind
+
+name: "Preferences"
+maturity: "merged"
+
+lineage: seqs: [
+	{
+		schemas: [
+			{//0.0
+				// UID for the home dashboard
+				homeDashboardUID?: string
+
+				// The timezone selection
+				// Would be nice it this used:
+				// import { TimeZone } from '@grafana/data';
+				timezone?: string
+
+				// day of the week (sunday, monday, etc)
+				weekStart?: string
+
+				// light, dark, empty is default
+				theme?: string
+
+				// Selected language (beta)
+				language?: string
+
+				navbar?: #NavbarPreference
+
+				queryHistory?: #QueryHistoryPreference
+
+				#NavLink: {
+					id: string
+					text?: string
+					URL?: string
+					target?: string
+				} @cuetsy(kind="interface")
+
+				#NavbarPreference: {
+					savedItems?: [...#NavLink]
```

Comment:
@joshhunt -- any idea if this is actually used?   I don't see any reference to it in the frontend code -- maybe it is just legacy and can be removed?

Reply:
The only references to the string "savedItems" is in our tests and old docs:

<img width="505" alt="image" src="https://user-images.githubusercontent.com/705951/205127778-4786c48b-655e-41c3-b711-192c9ae8db68.png">

Reply:
I removed it from the cue schema -- that was the most complicated part and was not yet implemented anyway ;)

Reply:
yep this was used to pin things in the megamenu but we decided against doing that. i guess we can rip it all out? 🤔